### PR TITLE
Arm backend: Add select operator

### PR DESCRIPTION
**Pull request by SaoirseARM**
Adds Select operator to the Arm backend including unit tests

### CODE REVIEW

File: backends/arm/operators/op_select.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+from executorch.backends.arm.tosa_mapping import TosaArg
+
+from executorch.backends.arm.tosa_utils import build_reshape, tosa_shape
+from serializer.tosa_serializer import TosaOp
+from torch.fx import Node
+
+
+@register_node_visitor
+class SelectVisitor(NodeVisitor):
+    target = "aten.select_copy.int"
```
Comment (digantdesai) [inline backends/arm/operators/op_select.py:23]: I guess we can't replace this with slice easily in a pass?

**Review by digantdesai — APPROVED** [review]

File: backends/arm/operators/op_select.py (hunk 1 above)
Comment (SaoirseARM) [reply backends/arm/operators/op_select.py:23]: Hi, 

Thanks for review!
We could use a pass for this, but there is some work that is on-going that should enable this going forward.
I hope this helps!

Thanks, 
Saoirse

**Pull request merged**